src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -118,7 +146,9 @@
     @SuppressWarnings("deprecation") // RunList.size
     @Test public void useBuildDiscarder() throws Exception {
         sampleRepo.init();
-        sampleRepo.write("Jenkinsfile", "properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '1']]])");
+        sampleRepo.write("Jenkinsfile", HAVE_SYMBOL ?
+            "properties([buildDiscarder(logRotator(numToKeepStr: '1'))])" :
```

Comment:
Maybe not directly related to this PR, but this array syntax is ugly. Can't we do blown-up closure like

```
properties {
  buildDiscarder(logRotator(numToKeepStr:1))
}
```

And also, we want more human readable overload for this guy, maybe something like

```
properties {
  discardBuild '2d'
}
```